src/pages/stacks-blockchain/sending-tokens.md

```diff
@@ -0,0 +1,207 @@
+
+## Step 1: Installing libraries
+
+First install the stacks transactions library, a JavaScript [BigNum](https://en.wikipedia.org/wiki/Arbitrary-precision_arithmetic) implementation, and the API client for the [Stacks 2.0 Blockchain API](/references/stacks-blockchain):
+
+```shell
+npm install --save @blockstack/stacks-transactions@0.6.0 bn.js @stacks/blockchain-api-client
+```
+
+## Step 2: Specifying a sender
+
+In order to build and sign transactions, you will need a Stacks privat key. You can easily generate a new, random Stacks 2.0 sender key (see ["Generating an account" from the previous tutorial](http://localhost:3000/stacks-blockchain/managing-accounts#step-2-generating-an-account)).
+
+For this tutorial, we will use an existing Stacks account and instantiate the key object from a private key string:
+
+```js
+const BN = require('bn.js');
+const {
+  makeSTXTokenTransfer,
+  createStacksPrivateKey,
+  broadcastTransaction,
+  estimateTransfer,
+  getNonce,
+  StacksTestnet,
+  privateKeyToString,
+} = require('@blockstack/stacks-transactions');
+const { TransactionsApi, Configuration } = require('@stacks/blockchain-api-client');
+
+const apiConfig = new Configuration({
+  fetchApi: fetch,
+  basePath: 'https://stacks-node-api-latest.argon.blockstack.xyz',
+});
+
+const key = 'edf9aee84d9b7abc145504dde6726c64f369d37ee34ded868fabd876c26570bc01';
+const senderKey = createStacksPrivateKey(key);
+```
+
+-> Note: The code above also imports methods required for the next steps, including API configuration for the client library usage.
+
+## Step 3: Generating transaction
+
+To generate a token transfer transaction, we will be using the `makeSTXTokenTransfer()` transaction builder function:
+
+```js
+const recipient = 'SP3FGQ8Z7JY9BWYZ5WM53E0M9NK7WHJF0691NZ159';
+
+// amount of Stacks tokens to send (in microstacks). 1,000,000 microstacks are worth 1 STX token
+const amount = new BN(1000000);
+
+// skip automatic fee estimation
+const fee = new BN(0);
+
+// skip automatic nonce lookup
+const nonce = new BN(0);
+
+// override default setting to broadcast to the Testnet network
+const network = new StacksTestnet();
+
+const memo = 'hello world';
+
+const txOptions = {
+  recipient,
+  amount,
+  fee,
+  nonce,
+  senderKey: privateKeyToString(senderKey),
+  network,
+  memo,
+};
+
+...
+
+const transaction = await makeSTXTokenTransfer(txOptions);
+```
+
+The generation method will need a few more pieces of information, as specified in the `txOptions` object:
+
+| Parameter   | Description                                                                                                                      | Optional |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `recipient` | The recipient Stacks address in c32check format                                                                                  | **No**   |
```

Comment:
[nit] I don't think we've described what `c32check` format means anywhere in this doc. Maybe link to https://www.npmjs.com/package/c32check?